src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+
+.main__container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0;
+  max-width: 1100px;
+  justify-content: center;
+  margin: 0 auto;
+}
```

Comment:
Move this code to a new scss file for main__container or catalog block(depends on class name).